source/extensions/quic_listeners/quiche/quic_filter_manager_connection_impl.cc

```diff
@@ -90,14 +102,22 @@ void QuicFilterManagerConnectionImpl::rawWrite(Buffer::Instance& /*data*/, bool
   NOT_REACHED_GCOVR_EXCL_LINE;
 }
 
+void QuicFilterManagerConnectionImpl::adjustBytesToSend(int64_t delta) {
+  bytes_to_send_ += delta;
```

Comment:
bytes_to_send_ is uint32_t but delta is int64. Can we make bytes_to_send_ 64 bit?

Reply:
SG